test/e2e/network_segmentation.go

```diff
@@ -1533,12 +1532,12 @@ func generateLayer3Subnets(cidrs string) []string {
 }
 
 func waitForUserDefinedNetworkReady(namespace, name string, timeout time.Duration) error {
```

Comment:
nit: also change the name of the utils to reflect its networkcreated?

Reply:
The intent is to have func name that describe CUDN is ready to work with regardless of which condition being asserted, make sense?